e2e-tests/specs/all.e2e.js

```diff
@@ -43,7 +43,7 @@ const config = {
     },
     featuredCards: ['c7d34f39-397c-3727-9dff-5d0d9d8cf731'],
     filterPanel: {
-        enabled: 'false',
+        enabled: 'true',
```

Comment:
@sheridansunier  "Carousel" layout will not work if filter panel enable. It is better to enable for filter tests.

<img width="783" alt="image" src="https://github.com/adobecom/caas/assets/124742619/58248835-85bb-448b-9973-ba8eb7f0eca7">

Reply:
that makes sense - but also the carousel has it's own separate configuration so I feel like it's fine?